sql/1.0xx/activity_feed.sql

```diff
@@ -0,0 +1,13 @@
+BEGIN;
+
+CREATE TABLE "activity_feed" (
+  "id" serial NOT NULL,
+  "created" timestamp with time zone NOT NULL,
+  "type" text NOT NULL,
```

Comment:
This is really simple which is awesome.  Another idea which is more complex is to use tags.  More flexible than hierarchical stuff and less string manipulation than this solution.

How to build a schema with tags (in case you are like me and didn't know off the top of your head):
http://skm.zoomquiet.io/data/20060423142114/index.html
http://stackoverflow.com/questions/20856/recommended-sql-database-design-for-tags-or-tagging